interfaces/automated-testing/scd/scd.yaml

```diff
@@ -57,4 +76,26 @@ paths:
         '404':
           description: The USS automated testing interface is not activated.
       summary: Retrieve the status of the USS automated testing interface
-      description: Get the status of the USS automated testing interface
\ No newline at end of file
+      description: Get the status of the USS automated testing interface
+
+  /inject/flight:
+    summary: Process operator details
```

Comment:
The summary on line 99 and description on line 100 are good, but I don't think that "Process operator details" is an accurate summary of the endpoint that will be `/v1/flights/{flight_id}` (nor the `/inject/flight` endpoint); I think we can just remove this summary line, allowing lines 99 & 100 to handle the documentation.

Reply:
Thanks this is now removed.